DigitalExcellence/dex-frontend
# Implemented all styling changes on Add New Project page.

## Description

Added the new designs of the "Project Add Page".

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open "Add Project" page in the nav bar
2. Check the designs 

## Link to issue

Closes: #188
